src/Checkbox/index.stories.js

```diff
@@ -38,6 +38,33 @@ export const MultipleCheckboxes = () => (
   </>
 );
 
+export const Indeterminate = () => {
+  const [isChecked, setIsChecked] = useState(false);
+  return (
+    <>
+      <Checkbox
+        label="Intermediate state"
```

Comment:
is this a typo? these words sound very similar

Reply:
Nice catch, thanks!